cmd/internal/client/apps.go

```diff
@@ -35,3 +51,51 @@ var CmdApps = &cobra.Command{
 	SilenceErrors: true,
 	SilenceUsage:  true,
 }
+
+// CmdAppsShow implements the carrier `apps show` command
+var CmdAppsShow = &cobra.Command{
```

Comment:
Nitpick: the var names and comments also still use the plural form.

Reply:
Done with commit 1823b167f79f1efa05871a307e3afdcb601cdf6b